src/data/set/disjointed.lean

```diff
@@ -17,6 +17,10 @@ variables {α : Type u} {β : Type v} {γ : Type w} {ι : Sort x}
 /-- A relation `p` holds pairwise if `p i j` for all `i ≠ j`. -/
 def pairwise {α : Type*} (p : α → α → Prop) := ∀i j, i ≠ j → p i j
 
+@[simp] theorem set.pairwise_on_univ {r : α → α → Prop} :
```

Comment:
I wouldn't set this one as a simp lemma: I think it is quite common to want to rewrite in the other direction to use lemmas on `pairwise_on` (at least, it happens pretty often for continuity or differentiability stuff).